src/include/list_link.h

```diff
@@ -78,12 +78,12 @@ typedef pbs_list_link pbs_list_head;
 
 /* function prototypes */
 
-extern void insert_link(pbs_list_link *old, pbs_list_link *new, void *pobj, int pos);
-extern void append_link(pbs_list_head *head, pbs_list_link *new, void *pnewobj);
+extern void insert_link(pbs_list_link *old, pbs_list_link *newp, void *pobj, int pos);
+extern void append_link(pbs_list_head *head, pbs_list_link *newp, void *pnewobj);
```

Comment:
What is the significance of renaming the new to newp? Both cases the parameter was a pointer anyway!